.github/workflows/release-testing.yaml

```diff
@@ -86,7 +86,6 @@ jobs:
         TEST_RUN: ${{ matrix.test }}
 
   e2e-tests:
-    timeout-minutes: ${{ fromJSON(vars.GHA_DEFAULT_TIMEOUT) }}
```

Comment:
Can you we set our own timeout instead?

Reply:
Yup, setting it to 30m as most of the runs oscillated between 10-25m.